portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/*
+ * Counts the number of times each city occurs in a list of markers. Returns {[city, count],...}
+ */
+function countMarkers(markers) {
+  const markerCounts = {};
+  for(marker of markers) {
```

Comment:
make this `const marker`. If you don't use const or let (or var) the variable becomes global. Probably not what you want.